crates/symbolicator/src/config.rs

```diff
@@ -431,4 +469,12 @@ mod tests {
         let cfg = Config::from_reader(yaml.as_bytes());
         assert!(cfg.is_ok());
     }
+
+    #[test]
+    fn test_empty_file() {
```

Comment:
Is this what we actually want to do? If an empty file is passed in, should we just use the defaults or is it too much of a pain to detect when the reader is empty?